docs/reference_guides/model_libraries/power_generation/unit_models/steam_valve.rst

```diff
@@ -19,7 +19,7 @@ Example
   from idaes.core import FlowsheetBlock
   from idaes.models_extra.power_generation.unit_models.helm import HelmValve
   from idaes.models.properties import iapws95
-  from idaes.ui.report import degrees_of_freedom, active_equalities
+  from idaes.core.ui.report import degrees_of_freedom, active_equalities
```

Comment:
I am surprised to see these being imported from the `ui` tools - are these the same as the functions from `idaes.core.util.model._statistics` and if so should they be imported from there??

Reply:
Great catch. idaes.ui don't have these functions. They are not being used in the given example, so I chose to remove that line from the documentation